src/check_filenames.m

```diff
@@ -2,7 +2,7 @@
 
 if ~isempty(file_list)
     
-    csv_files = sort(file_list(cell_contains(file_list, '_emg.csv')));
```

Comment:
We can leave with it, but by stating the file naming, the code could not work if we change for any reason the filename structure. Providing the file explicitly as input file should be enough, placing the responsibility on the user side to make that : 1st file is emg for 2kmh, 2nd file is gait event for 2kmh, 3rd is .... etc. Don't you think?